composer.json

```diff
@@ -36,11 +36,13 @@
     },
     "autoload": {
         "psr-4": {
-            "Misd\\PhoneNumberBundle\\": ""
-        },
-        "exclude-from-classmap": [
-            "/Tests/"
-        ]
+            "Misd\\PhoneNumberBundle\\": "src/"
+        }
+    },
+    "autoload-dev": {
+        "psr-4": {
+            "Misd\\PhoneNumberBundle\\Tests\\": "tests/"
```

Comment:
suffixing tests with a `Test` namespace is a "meh" practice at best. It's better imo to have the tests in the same namespace.

Reply:
I'am not agree for this point, as the composer doc says :
`Classes needed to run the test suite should not be included in the main autoload rules to avoid polluting the autoloader in production and when other people use your package as a dependency.
`

Reply:
There's s light difference here. It does not say "not to use the same namespace". Just "not the same autoloading rules", which is fine then in the `autoload-dev` section.

Reply:
Yes but we need to differ them

Reply:
No. 

```json
{
	"autoload": {
		"psr-4": {
			"App\\": "src"
		
		}
	},

	"autoload-dev": {
		"psr-4": {
			"App\\": "tests"
		}
	}
}
```

works wonder. When doing a `composer install` from the root, then `App\\` prefix will load from both `src` and `tests`. When using `--no-dev`, it will only look in `src`, the same when this package isn't the root package.

Reply:
Good to know, thanks :)
WDYT @odolbeau ?

Reply:
I'm used to suffix the test namespace with `Test` also but TBH, I don't know why! :D
I don't have any opinion on this, both are fine.

It's not linked to this PR anyway. :)